core/src/main/java/eu/fasten/core/data/Constants.java

```diff
@@ -12,6 +12,8 @@
 
     public static final String debianForge = "debian";
 
+    public static final String pypiForge = "PyPI";
```

Comment:
@vitsalis Could you please keep one of pypiForge or pyForge, and update the code base accordingly?